src/main/java/com/cburch/logisim/fpga/hdlgenerator/HDL.java

```diff
@@ -83,22 +83,22 @@ public static String notEqualOperator() {
 
   public static String greaterOperator(String signalOne, String signalTwo, boolean signed, boolean equal) {
     return (isVHDL()
-                ? "STD_LOGIC_VECTOR("+ (signed ? "SIGNED" : "UNSIGNED") +"("+ signalOne +")"
-                : (signed ? "$signed("+ signalOne +")" : signalOne))
+                ? "STD_LOGIC_VECTOR(" + (signed ? "SIGNED" : "UNSIGNED") + "(" + signalOne + ")"
```

Comment:
This `"STD_LOGIC_VECTOR(" + (signed ? "SIGNED" : "UNSIGNED") + "(" + signalOne + ")"` line is repeated in multiple places. Can you make private function that accepts `signalOne` and `signed` as arguments and returns `String`? Same for ":" part of ternary